common.cmake

```diff
@@ -10,20 +10,32 @@ if(__NRFXLIB_COMMON_CMAKE__)
 endif()
 set(__NRFXLIB_COMMON_CMAKE__ TRUE)
 
+#
+# Build the path of the library taking in
+# -Arch type
+# -Float ABI
+# -wchar_t size
+#
 function(nrfxlib_calculate_lib_path lib_path)
+  # Add Arch type
+  assert(GCC_M_CPU "GCC_M_CPU must be set to find correct lib.")
+  set(temp_path lib/${GCC_M_CPU}/)
+  # Append floating ABI
   if(CONFIG_FLOAT)
 	if(CONFIG_FP_HARDABI)
       set(float_dir hard-float)
 	elseif(CONFIG_FP_SOFTABI)
       set(float_dir softfp-float)
 	else()
       assert(0 "Unreachable code")
-	endif()
+  endif()
   else()
 	set(float_dir soft-float)
   endif()
-
-  assert(GCC_M_CPU "GCC_M_CPU must be set to find correct lib.")
-
-  set(${lib_path} lib/${GCC_M_CPU}/${float_dir} PARENT_SCOPE)
```

Comment:
It is cleaner to do

`set(${lib_path} lib/${GCC_M_CPU}/${float_dir}${wchar_dir} PARENT_SCOPE)
`

Note that there is intentionally no / between float_dir and wchar_dir (because wchar_dir would include it when enabled).

Then we can still read how the path looks in general.

Reply:
Fixed